fvtest/compilertriltest/ILValidatorTest.cpp

```diff
@@ -46,6 +45,12 @@ INSTANTIATE_TEST_CASE_P(ILValidatorTest, IllformedTrees, ::testing::Values(
     "(method return=Address (block (areturn (aladd (aconst 4) (iconst 1)))))",
     "(method return=Address (block (areturn (aiadd (iconst 1) (aconst 4)))))",
     "(method return=Address (block (areturn (aladd (lconst 1) (aconst 4)))))",
+#ifdef OMR_ENV_DATA64
+    "(method return=Address (block (areturn (aiadd (aconst 1) (iconst 4)))))",
+#endif
+#ifdef OMR_ENV_DATA32
+    "(method return=Address (block (areturn (aladd (aconst 1) (lconst 4)))))",
+#endif
```

Comment:
If I had to guess xlC is choking on this because `INSTANTIATE_TEST_CASE_P` is also a macro and it gets processed first. I would preprocess this file and check what this thing ends up expanding to. We may have to do some weird workaround for xlC unless someone has ideas.